paramiko/sftp_client.py

```diff
@@ -773,6 +780,10 @@ def getfo(self, remotepath, fl, callback=None, prefetch=True):
             the bytes transferred so far and the total bytes to be transferred
         :param bool prefetch:
             controls whether prefetching is performed (default: True)
+        :param int max_outstanding_prefetch_requests:
+            Limit the number of outstanding requests instead of sending all
```

Comment:
Is this parameter defining the maximum _outstanding_ requests, or the max _concurrent_ requests? It seems more the latter to me.

Separately, this parameter is an `int` -- its description should explicitly say how that value is used. It's _implicit_ that the value is the max number of concurrent requests, but it should be made explicit.

Reply:
I meant outstanding in the sense of "not yet done": the requests were sent, but we have not yet received an answer. But concurrent is also fine! :slightly_smiling_face:

Reply:
@kschoelhorn Yeah, that was what prompted my suggestion: to my mind, 'outstanding' could be ambiguous, meaning either:

1. The number of requests currently in-flight (the intended meaning)
2. The total number of requests yet to be completed, including both in-flight and not-yet-sent requests (not the intended meaning)

Meaning (2) doesn't make a whole lot of sense, practically, but if that's the sense of the word that someone reaches first, then the name could be confusing. Thus, my recommendation for 'concurrent'.